spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -267,8 +267,8 @@
         value: VMware Tools
     context_type: hash
     '
-    expect(filter.to_ruby).to eq("<value type=string>guest_applications.name</value> == \"VMware Tools\"")
-    expect(Condition.subst(filter.to_ruby, data, {})).to eq("\"VMware Tools\" == \"VMware Tools\"")
+    expect(filter.to_ruby).to eq("<value type=string>guest_applications.name</value> == \"vmware tools\"")
```

Comment:
I think this is ok because this is testing the conversion of the expression to ruby, not the evaluation. So, with the change to downcase both sides of a string comparison, this test needed to be changed.